internal/view/helm.go

```diff
@@ -38,5 +40,23 @@ func (c *Helm) bindKeys(aa ui.KeyActions) {
 		ui.KeyShiftN: ui.NewKeyAction("Sort Name", c.GetTable().SortColCmd(nameCol, true), false),
 		ui.KeyShiftS: ui.NewKeyAction("Sort Status", c.GetTable().SortColCmd(statusCol, true), false),
 		ui.KeyShiftA: ui.NewKeyAction("Sort Age", c.GetTable().SortColCmd(ageCol, true), false),
+		ui.KeyV:      ui.NewKeyAction("Values Get", c.getValsCmd(false), true),
+		ui.KeyA:      ui.NewKeyAction("Values Get All", c.getValsCmd(true), true),
 	})
 }
+
+func (c *Helm) getHelmDao() *dao.Helm {
+	return model.Registry["helm"].DAO.(*dao.Helm)
+}
+
+func (c *Helm) getValsCmd(allValues bool) func(evt *tcell.EventKey) *tcell.EventKey {
+	return func(evt *tcell.EventKey) *tcell.EventKey {
+		path := c.GetTable().GetSelectedItem()
```

Comment:
Need to check the path here! ie if nothing is selected no point on going any further.